### ISSUE TITLE
New Tab Tools is now available for Google Chrome

### ISSUE THREAD

**Issue by darktrojan** [darktrojan/newtabtools#325]
With all the effort it took to make New Tab Tools into a WebExtension, it seemed silly not to have it on Chrome too. Now you can:

https://chrome.google.com/webstore/detail/new-tab-tools/gfonmeedigdbjpgpeojioadpimflbogh

Enjoy.

**Comment by DanaMW** [comment]
Forgive me but didnt you already have it there? I loaded both chromes with the URL you pasted a week or so ago. Hope that was yours,

**Comment by darktrojan** [comment]
Yes I did, and now the URL is more public than before. Also I've changed the listing so that it appears in the Chrome store.

**Comment by sdean7855** [comment]
Is there a way to import my old tiles from FFx?  Mind you, I can't start NTT on FFx.  I have tried everything (all 4 of them) suggested, no joy

**Comment by darktrojan** [comment]
No, @sdean7855, it's the same as the Firefox version, and as such doesn't have the import/export parts yet. Try a fresh profile if you can't get NTT to start in Firefox.

**Comment by Kogl1n** [comment]
I haven't used the WebEx for Firefox, but the old FF addon had the thumbnails appear instantly when opening a new tab. In Chrome it is much slower, especially if you have used only one specific tab for quite a long time.
